packages/lpa-questionnaire-web-app/src/views/development-plan.njk

```diff
@@ -0,0 +1,121 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "components/appeal-details/macro.njk" import appealDetails %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: backLink,
```

Comment:
@davehlewis Excuse my ignorance, is this the right way to still do a backlink?

Reply:
Based from Dave's backlink.

Reply:
Yep that's the right way. backLink is a session variable set by the appropriate page. It falls back to task list page. While not necessarily the "right" way, it is "a" way that works right now